ee/tables/execparsers/flatpak/remote_ls/upgradeable/parser.go

```diff
@@ -0,0 +1,43 @@
+package flatpak_upgradeable
+
+import (
+	"bufio"
+	"io"
+	"regexp"
+)
+
+// The app id conforms to: [dbus specification](https://dbus.freedesktop.org/doc/dbus-specification.html#message-protocol-names)
+var flatpakAppIdRegexp = regexp.MustCompile(`(([a-zA-Z0-9_]+\.){1,}([a-zA-Z0-9_]+))`)
+
+// Wow this has been a head-pounding bugger of a thing.
+//
+// flatpak remote-ls --updates
+// data is separated by whitespace only
+// data headers aren't in stdout, they are written directly to TTY
```

Comment:
OMG really?

Reply:
You know... we used to have an exec tool that faked a pty. We deleted it a bit ago, but https://github.com/kolide/launcher/tree/19cb2b48a4df91021132ab5b62dadbff79a74def/pkg/ptycmd

Is it worth looking at bringing that back for headers and then doing something straightforward?

Reply:
I did go down the path of attempting to read the headers from tty, but I spent a while without making progress, so I stepped away from that solution. If there's prior art for this, then it's worth trying out.

Reply:
Tried going down this list, but I wasn't able to get any output in my testing, so I'm going to merge this for now and revisit as needed.